example/app.py

```diff
@@ -6,10 +6,21 @@
 PORT = 5000
 app = Flask(__name__)
 
-app.config.update({'SERVER_NAME': 'localhost:{}'.format(PORT),
-                   'SECRET_KEY': 'dev_key'})
-auth = OIDCAuthentication(app, issuer="https://localhost:50009")
-
+# See http://flask.pocoo.org/docs/0.12/config/
+app.config.update({'SERVER_NAME': 'example.com',
+                   'SECRET_KEY': 'dev_key',
+                   'PREFERRED_URL_SCHEME': 'https',
+                   'SESSION_PERMANENT': True, #turn on flask session support
+                   'PERMANENT_SESSION_LIFETIME': 2592000, #session time in seconds (30 days)
+                   'DEBUG': True})
+
+client_info = {
```

Comment:
I prefer to keep the example app as simple as possible.
The documentation in the README should be enough for anyone wanting to enable this feature, please remove this added config for the `session_refresh_interval`.